docs/content/en/docs/install/install.md

```diff
@@ -179,18 +201,37 @@ and update your configuration.
 Conversely, you can exclude some components when you install Keptn
 then later add them in.
 
-### Enable Keptn Lifecycle Operator (Observability)
+### Enable Keptn Lifecycle Operator (Observability and/or Release Lifecycle Management)
 
-If you only want to run the Keptn Observability use-case in your cluster,
+If you only want to run the Keptn Observability
+and/or Release Lifecycle Management use-cases in your cluster,
 you do not need to install the Keptn Metrics Operator.
-To disable it, modify the `keptn/values.yaml` file like this:
+To disable it, set the `metricsOperator` parameter
+in the `keptn/values.yaml` file to `disabled` as in the following:
 
 {{< embed path="/docs/content/en/docs/install/assets/values-only-lifecycle.yaml" >}}
 
-Keptn Observability must be enabled by
-[Namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
-To enable Observability in your namespace, you need to annotate it.
-For example, for the `simplenode-dev` namespace:
+Note that, if you want to run pre- and/or post-deployment
+[evaluations](../implementing/evaluations.md)
+as part of the Release Lifecycle Management use-case,
+you need to run the Keptn Metrics Operator.
+
+You must also enable Keptn for each
+[namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+on which you want to run either
+the Observability or Release Lifecycle Management use-case.
+This is because Keptn communicates with the Kubernetes scheduling mechanism
+for tasks such as enforcing checks natively,
+stopping a deployment from proceeding when criteria are not met,
+doing post-deployment evaluations
+and tracing all activities of all deployment
+[workloads](https://kubernetes.io/docs/concepts/workloads/)
+on the cluster.
+
+To enable Keptn in your namespace,
+annotate the appropriate `Namespace` resource(s).
+For example, for the `simplenode-dev` namespace,
+the annotation looks like this:
 
 ```yaml
```

Comment:
this yaml could be extracted into a standalone file.

Reply:
I don't understand your comment here.

Reply:
you should create a separate yaml file with this content and use `embed path` to reference it

Reply:
Sorry, it was changed on my local branch and not yet pushed.  It's gone now.